.github/CODEOWNERS

```diff
@@ -71,24 +71,27 @@ crates/external_services/src/grpc_client/dynamic_routing.rs @juspay/hyperswitch-
 crates/external_services/src/grpc_client/dynamic_routing/ @juspay/hyperswitch-routing
 crates/external_services/src/grpc_client/health_check_client.rs @juspay/hyperswitch-routing
 
-crates/api_models/src/payment_methods.rs @juspay/hyperswitch-routing
-crates/router/src/core/payment_methods.rs @juspay/hyperswitch-routing
-crates/router/src/routes/payment_methods.rs @juspay/hyperswitch-routing
-crates/router/src/types/api/payment_methods.rs @juspay/hyperswitch-routing
-crates/router/src/types/storage/payment_method.rs @juspay/hyperswitch-routing
-crates/router/src/db/payment_method.rs @juspay/hyperswitch-routing
-crates/diesel_models/src/payment_method.rs @juspay/hyperswitch-routing
-crates/diesel_models/src/query/payment_method.rs @juspay/hyperswitch-routing
-crates/storage_impl/src/payment_method.rs @juspay/hyperswitch-routing
-crates/openapi/src/routes/payment_method.rs @juspay/hyperswitch-routing
-crates/router/src/core/payment_methods/cards.rs @juspay/hyperswitch-routing
-crates/router/src/core/payment_methods/utils.rs @juspay/hyperswitch-routing
-crates/router/src/core/payment_methods/transformers.rs @juspay/hyperswitch-routing
-crates/router/src/core/payment_methods/vault.rs @juspay/hyperswitch-routing
-crates/router/src/core/payment_methods/migration.rs @juspay/hyperswitch-routing
-crates/router/src/core/payment_methods/surcharge_decision_configs.rs @juspay/hyperswitch-routing
-crates/router/src/workflows/payment_method_status_update.rs @juspay/hyperswitch-routing
-crates/router/src/core/payments/tokenization.rs @juspay/hyperswitch-routing
+crates/api_models/src/payment_methods.rs @juspay/hyperswitch-payment-methods
+crates/router/src/core/payment_methods.rs @juspay/hyperswitch-payment-methods
+crates/router/src/routes/payment_methods.rs @juspay/hyperswitch-payment-methods
+crates/router/src/types/api/payment_methods.rs @juspay/hyperswitch-payment-methods
+crates/router/src/types/storage/payment_method.rs @juspay/hyperswitch-payment-methods
+crates/router/src/db/payment_method.rs @juspay/hyperswitch-payment-methods
+crates/diesel_models/src/payment_method.rs @juspay/hyperswitch-payment-methods
+crates/diesel_models/src/query/payment_method.rs @juspay/hyperswitch-payment-methods
+crates/storage_impl/src/payment_method.rs @juspay/hyperswitch-payment-methods
+crates/openapi/src/routes/payment_method.rs @juspay/hyperswitch-payment-methods
+crates/router/src/core/payment_methods/cards.rs @juspay/hyperswitch-payment-methods
+crates/router/src/core/payment_methods/utils.rs @juspay/hyperswitch-payment-methods
+crates/router/src/core/payment_methods/transformers.rs @juspay/hyperswitch-payment-methods
+crates/router/src/core/payment_methods/vault.rs @juspay/hyperswitch-payment-methods
+crates/router/src/core/payment_methods/migration.rs @juspay/hyperswitch-payment-methods
+crates/router/src/core/payment_methods/surcharge_decision_configs.rs @juspay/hyperswitch-payment-methods
```

Comment:
You can consider including the entire `crates/router/src/core/payment_methods` payment methods directory if that helps.